INSTALL.md

```diff
@@ -63,10 +63,19 @@ DocumentRoot /path/to/friendica-directory/public/
 ```
 
 ### Nginx
-Include this line your nginx config file.
+Add these lines to your nginx config file.
 
 ```
 root /path/to/friendica-directory/public;
+
+location / {
+    rewrite ^/(.*) /index.php?$args last;
+}
+
+location ~* \.(jpg|jpeg|gif|png|ico|css|js|htm|html|ttf|woff|svg)$ {
+    expires 30d;
+    try_files $uri /index.php?q=$uri&$args;
```

Comment:
Are you sure you need the `q=$uri` here?

Reply:
Not required. I removed it.

```suggestion
    try_files $uri /index.php?$args;
```

What about these `ttf|woff|svg` ?

Reply:
We use fontawesome so I would keep the mentions of the fonts and svg files.

Reply:
OK. Thanks.